core/merkle-patricia-trie/src/db.rs

```diff
@@ -0,0 +1,54 @@
+
+use hashbrown::HashMap;
+use crate::std::*;
+
+#[derive(Debug)]
+pub struct MemoryDB {
+    data: RefCell<HashMap<Vec<u8>, Vec<u8>>>
+}
+
+impl MemoryDB {
```

Comment:
recommend add `#[inline]` for these tiny & light functions

Reply:
Using of ```#[inline]``` seems to be controversial here? Here is some discussion on this 
https://internals.rust-lang.org/t/when-should-i-use-inline/598/3

This crate is is strongly inspired by https://github.com/citahub/cita-trie , adding some proof related codes.

Reply:
https://stackoverflow.com/questions/37639276/when-should-inline-be-used-in-rust

Ok, ignore for the time being.